packages/stencil/src/components/datacom-list/datacom-list.tsx

```diff
@@ -20,16 +20,21 @@ export class DatacomList {
 
   render() {
     const classes = {
-      [`dc-list-${this.variant}`]: true,
+      'dc-list-ordered': true,
       [`dc-list-${this.type}`]: true,
     };
 
-    const ListElement = this.variant === 'ordered' ? 'ol' : 'ul';
     return (
       <Host>
-        <ListElement class={classes}>
-          <slot></slot>
-        </ListElement>
+        {this.variant == 'ordered' ? (
```

Comment:
You didn't need to change all this.

You could have just updated the conditional class on line 24 to only apply when the ordered variant is selected. (Which is the same conditional you previously used for your `LineElement` const before.)